### ISSUE TITLE
Please add recipes for "Diphenhydramine" and "Histamine"

### ISSUE THREAD

**Issue by Ss14plss** [space-wizards/space-station-14#23000]
Hello, I kindly ask you to add recipes or a method of extraction for histamine and Diphenhydramine, this will greatly help doctors.

**Comment by Holinka4ever** [comment]
Just for sure,  Diphenhydramine is a posion medicine healing that heals 2 poison damage more than dyloven.
Histamine is just allergic thing, you can get it from uh emagges booze dispenser. You can also make a medicine from it, Ultravasculine, that heals toxic damage, instead of specifec poison. But it deals brute damage when used, overdose just make damage bigger.

**Comment by Ss14plss** [comment]
> Наверняка, Дифенгидрамин — это лекарство от яда, которое лечит на 2 урона ядом больше, чем диловен. Гистамин вызывает аллергию, его можно получить из дозатора выпивки. Вы также можете сделать из него лекарство, Ультраваскулин, которое лечит токсические повреждения вместо конкретного яда. Но при использовании он наносит грубый урон, передозировка только увеличивает урон.

Can you tell us a little more about histamine production? Thank you

**Comment by IamVelcroboy** [comment]
We added anti-histamine on DV. Histamine would probably be something you extract from plants.

![Screenshots_2023-12-26-07-07-25](https://github.com/space-wizards/space-station-14/assets/107660393/4f8044ec-bad8-49ea-a2c1-cfabcd7fab83)

**Comment by IamVelcroboy** [comment]
Here, I had to get some coffee first and wake up a bit but I copied it over. [#23006](https://github.com/space-wizards/space-station-14/pull/23006) 

You can extract histamine from nettles already.  Not sure what else would potentially produce it but there is currently at least the one source.

**Comment by IProduceWidgets** [comment]
You can also get them from various science things.

**Comment by hord-brayden** [comment]
I agree a recipe should be available, additionally I think histamine would be very interesting if it were implemented in more thorough way throughout. Just one example; [histamine can be released from injury/damage ](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7012972/#:~:text=In%20the%20PNS%2C%20histamine%20is,the%20generation%20of%20pain%20hypersensitivity.)which also induces and insinuates that pain is present.  It may be an interesting mechanic, if pain is ever a thing along with morphine etc, to keep histamine's OD levels in addition to having it as a recipe and generating from pain as well as other sources? Just a thought, as it pertains to balance..
